workflows/publish-imagemosaic.json

```diff
@@ -5,19 +5,28 @@
         "type": "download-file",
         "inputs": [
             "http://nginx/sample.tif",
-            "/opt/geoserver_data/sample.tif"
+            "/opt/geoserver_data/20220101T0000.tif"
```

Comment:
please add that file to https://github.com/klips-project/klips-sdi/tree/main/mocked-webspace

Reply:
no need i think. sample.tif will be downloaded and saved as `20220101T0000.tif`.

Reply:
please confirm.

Reply:
haha ok, tomatoes on the eyes... :+1: